pangeo-cnes.tex

```diff
@@ -83,75 +99,94 @@ \subsection{Motivations, mission and goals}
+\item Jupyter: Jupyter notebooks and Jupyter Lab enable interactive computing and analysis from a web browser, and JupyterHub adds multi-user support.  Jupyter notebooks are quickly becoming the standard open-source format for interactive computing not only in Python, but also in languages such as Julia and R.
 \end{itemize}
 
-The GitHub community offers online documentation, scripts and other tools to link them together in order to deploy a Pangeo platform (see \ref{pangeo_stack}) and put the software stack on HPC systems or in the public cloud. The main elements allowing to build and use the platforms along the core libraries are first a set of scripts and documentation that allows automatically creating the necessary cloud infrastructure. Currently available for Google Cloud Engine (GCE), and work is underway for Amazon Web Services (AWS) compatibility. Terraform software and CI/CD tooling are also under active development and offer the potential for automating these tasks.
-
-Modules to automatically deploy distributed clusters in various infrastructures are being developed: dask-kubernetes for Kubernetes clusters and thus the public cloud, dask-jobqueue\cite{b4} for HPC systems using scheduler such as Slurm, PBS Pro or LSF, and dask-yarn for YARN clusters (i.e. Hadoop).
-
 \begin{figure}
   \centering
   \includegraphics[width=\columnwidth]{pangeo_stack.png}
   \caption{\label{pangeo_stack} Pangeo platform main components.}
 \end{figure}
 
+There are several developments that either started in or are fueled by the
+Pangeo community:
+\begin{itemize}
+\item Modules to automatically deploy Dask distributed clusters in various infrastructures are being developed: dask-kubernetes for Kubernetes clusters and thus for the public cloud, dask-jobqueue\cite{b4} for HPC systems using scheduler such as Slurm, PBS Pro or LSF, and dask-yarn for YARN clusters (i.e. Hadoop).
+\item The integration of a Zarr backend in xarray paved the way to directly access cloud-based object storage in parallel computations.
+\item ...
+\end{itemize}
+
+
+\subsection{Deployments}
+\label{ssec:deployments}
+
+The GitHub community offers online documentation, scripts and other tools to link them together in order to deploy a Pangeo platform (see \ref{pangeo_stack}) and put the software stack on HPC systems or in the public cloud. The main elements allowing to build and use the platforms along the core libraries are first a set of scripts and documentation that allows automatically creating the necessary cloud infrastructure. Currently available for Google Cloud Engine (GCE), and work is underway for Amazon Web Services (AWS) compatibility. Terraform software and CI/CD tooling are also under active development and offer the potential for automating these tasks.
```

Comment:
Does the `Deployments` subsection needs rewriting?
I have the feeling it could be clearer and sharper.

Reply:
If you can propose something, I'm all hears :)! I will try to merge this soon, feel free to open another PR!